### ISSUE TITLE
exclude_from_weight_decay in RectifiedAdam()

### ISSUE THREAD

**Issue by markub3327** [tensorflow/addons#2681]
**Describe the feature and the current behavior/state.**

Can `RectifiedAdam` have same parameter `exclude_from_weight_decay` as `AdamW`?  We must exclude some parameters of the model from weight decay.

**Relevant information**
- Are you willing to contribute it (yes/no): yes
*If you wish to contribute, then read the requirements for new contributions in [`CONTRIBUTING.md`](https://github.com/tensorflow/addons/blob/master/CONTRIBUTING.md#requirements-for-new-contributions-to-the-repository)*
- Are you willing to maintain it going forward? (yes/no): yes
- Is there a relevant academic paper? (if so, where): no
- Does the relavent academic paper exceed 50 citations? (yes/no): no
- Is there already an implementation in another framework? (if so, where): yes
- Was it part of tf.contrib? (if so, where): No

**Which API type would this fall under (layer, metric, optimizer, etc.)**

optimizer

**Who will benefit with this feature?**

Everyone

**Any other info.**

Thanks a lot.

**Comment by bhack** [comment]
Probably this component it is orphan. 
Gently ping @szutenberg

@markub3327 Can you check with a ticket on https://github.com/keras-team/keras if they are interested if you port this in Keras with your `exclude_from_weight_decay` extension?

**Comment by markub3327** [comment]
@bhack  The ticket is done.

**Comment by bhack** [comment]
Thank you as `AdamW` was just duplicated [there 25 days ago](https://github.com/keras-team/keras/commit/3b87e6c50be9cd8618aab5b154a72c20a6b97f9b) probably you could add a comment there also for `exclude_from_weight_decay` for `AdamW`

**Comment by markub3327** [comment]
@bhack I have done it.

Please look [here](https://github.com/keras-team/keras/issues/16201).

**Comment by seanpmorgan** [comment]
TensorFlow Addons is transitioning to a minimal maintenance and release mode. New features will not be added to this repository. For more information, please see our public messaging on this decision:
[TensorFlow Addons Wind Down](https://github.com/tensorflow/addons/issues/2807)

Please consider sending feature requests / contributions to other repositories in the TF community with a similar charters to TFA:
[Keras](https://github.com/keras-team/keras)
[Keras-CV](https://github.com/keras-team/keras-cv)
[Keras-NLP](https://github.com/keras-team/keras-nlp)
